src/screens/landingPageScreens/ApplicationInfoScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import BlankImg from "/images/Landing Page Images/BlankImage.png";
+
+interface CardData {
```

Comment:
CardData is redundand. consider creating an util interface in src/data/utils or smth what you find appropriate

interface CardData {
    title: string;
    description: string;
}